app/components/dashboard/hypertension/overdue_patients_called_table_component.html.erb

```diff
@@ -0,0 +1,82 @@
+<div class="card mb-16px mt-0 w-full">
+  <%= render Dashboard::Card::TitleComponent.new(
+    title: "Overdue patients called",
+    subtitle: t("overdue_patients_call_activity.reports_card_subtitle") 
+  ) do |c| %>
+    <%= c.tooltip({ "Patients called" => t("overdue_patients_call_activity.patients_called"),
+                    "Percent called" => "",
```

Comment:
Empty key

Reply:
This it because its acting as a title 

![Uploading Screenshot 2023-06-08 at 10.31.28.png…]()

Reply:
Oh okay. We can refactor the tooltip component to accommodate for this as well, once the feature is released.